actors/builtin/miner/expiration_queue_test.go

```diff
@@ -597,8 +598,8 @@ func TestExpirationQueue(t *testing.T) {
 		assertBitfieldEquals(t, removed.OnTimeSectors, 1, 4, 6)
 		assertBitfieldEquals(t, removed.EarlySectors, 5)
 		assert.Equal(t, abi.NewTokenAmount(1000+1003+1005), removed.OnTimePledge) // only on-time sectors
-		assert.True(t, removed.ActivePower.Equals(PowerForSectors(sectorSize, []*SectorOnChainInfo{sectors[0], sectors[5]})))
-		assert.True(t, removed.FaultyPower.Equals(PowerForSectors(sectorSize, sectors[3:5])))
+		assert.True(t, removed.ActivePower.Equals(PowerForSectors(sectorSize, []*SectorOnChainInfo{sectors[0]})))
+		assert.True(t, removed.FaultyPower.Equals(PowerForSectors(sectorSize, sectors[3:6])))
```

Comment:
@acruikshank did I do this right?

Reply:
What you did makes sense to me in that a recovered sector is still a fault. I think it's right that recovered sector's power is still faulty. In that case, your changes are strictly better than what came before. 

It might be worth adding an assertion that recovering needs to be a subset of faults or at least documenting that expectation. I'm probably going to be back in here, so I can do it.

Reply:
Makes sense. It shouldn't be too expensive and it's an invariant we want to maintain.